spinalcordtoolbox/scripts/sct_propseg.py

```diff
@@ -662,6 +662,11 @@ def propseg(img_input, options_dict):
 
 def main(argv=None):
     parser = get_parser()
+    if sys.platform.startswith("win32"):
+        # This isn't *really* a parsing error, but it feels a little more official to display the help with this error
+        parser.error("`sct_propseg` is not currently supported on native Windows installations. \n\n"
+                     "For spinal cord segmentation, please migrate to the new and improved `sct_deepseg_sc` tool, "
+                     "or consider using WSL to install SCT instead.")
```

Comment:
Getting `sct_propseg` to work on Windows seems like a TODO type of thing, could we open an issue about it? Even if the ultimate decision is to close the issue as WONTFIX, we could include a link to it here and people would get the latest news about it.

Reply:
Good thinking! In that case, it may be best to keep the existing linked issue open, because it already contains a bit of a summary of the existing build issues.

I'll add a note about this to the error message, too. :)